idl/rrdb.thrift

```diff
@@ -147,6 +147,29 @@ struct multi_get_response
     6:string        server;
 }
 
+struct batch_get_request {
```

Comment:
After we reach an agreement to the API name, I think it's better to keep these thrift structure names to be the same/similar to the API's name too. Of course, the server side should be updated too.

Reply:
I think api name need not like inner rpc request struct name since the batchGet2 api interface used very different named innser rpc request struct;   in my opinion api name has nothing to do with the inner rpc equest name ; 
  'batch_get_request' is more simpler and can be easily differentiated from "single get  request" and "multi get request"; that is enough~